float32 pos
float32 vel
float32 kp
float32 kd
float32 tau_ff